src/20240603_Multi_MayFundingUpdate/AaveV3Ethereum_MayFundingUpdate_20240603.sol

```diff
@@ -0,0 +1,323 @@
+      AaveV2EthereumAssets.USDC_A_TOKEN,
+      address(this),
+      IERC20(AaveV2EthereumAssets.USDC_A_TOKEN).balanceOf(address(AaveV3Ethereum.COLLECTOR)) - 1e6
+    );
+
+    AaveV3Ethereum.COLLECTOR.transfer(
+      AaveV3EthereumAssets.USDC_A_TOKEN,
+      address(this),
+      AETH_USDC_AMOUNT
+    );
+
+    AaveV2Ethereum.POOL.withdraw(
+      AaveV2EthereumAssets.USDC_UNDERLYING,
+      type(uint256).max,
+      address(SWAPPER)
+    );
+
+    AaveV3Ethereum.POOL.withdraw(
+      AaveV3EthereumAssets.USDC_UNDERLYING,
+      type(uint256).max,
+      address(SWAPPER)
+    );
+
+    uint256 usdcBalance = IERC20(AaveV3EthereumAssets.USDC_UNDERLYING).balanceOf(address(SWAPPER));
+
+    SWAPPER.swap(
+      MILKMAN,
+      PRICE_CHECKER,
+      AaveV3EthereumAssets.USDC_UNDERLYING,
+      AaveV3EthereumAssets.GHO_UNDERLYING,
+      USDC_FEED,
+      GHO_USD_FEED,
+      address(AaveV3Ethereum.COLLECTOR),
+      usdcBalance,
+      50
+    );
+
+    /// DPI
+    uint256 dpiBalance = IERC20(AaveV2EthereumAssets.DPI_UNDERLYING).balanceOf(
+      address(AaveV3Ethereum.COLLECTOR)
+    );
+
+    AaveV3Ethereum.COLLECTOR.transfer(
+      AaveV2EthereumAssets.DPI_UNDERLYING,
+      address(SWAPPER),
+      dpiBalance
+    );
+
+    SWAPPER.swap(
+      MILKMAN,
+      PRICE_CHECKER,
+      AaveV2EthereumAssets.DPI_UNDERLYING,
+      AaveV3EthereumAssets.GHO_UNDERLYING,
+      DPI_FEED,
+      GHO_USD_FEED,
+      address(AaveV3Ethereum.COLLECTOR),
+      dpiBalance,
+      500
+    );
+
+    /// aEthUSDT
+    AaveV3Ethereum.COLLECTOR.transfer(
+      AaveV3EthereumAssets.USDT_A_TOKEN,
+      address(this),
+      AETH_USDT_AMOUNT
+    );
+
+    AaveV3Ethereum.POOL.withdraw(
+      AaveV3EthereumAssets.USDT_UNDERLYING,
+      type(uint256).max,
+      address(SWAPPER)
+    );
+
+    uint256 usdtBalance = IERC20(AaveV3EthereumAssets.USDT_UNDERLYING).balanceOf(address(SWAPPER));
+
+    SWAPPER.swap(
+      MILKMAN,
+      PRICE_CHECKER,
+      AaveV3EthereumAssets.USDT_UNDERLYING,
+      AaveV3EthereumAssets.GHO_UNDERLYING,
+      AaveV3EthereumAssets.USDT_ORACLE,
+      GHO_USD_FEED,
+      address(AaveV3Ethereum.COLLECTOR),
+      usdtBalance,
+      50
+    );
+  }
+
+  function _unwrapLsts() internal {
+    uint256 rethBalance = IERC20(AaveV3EthereumAssets.rETH_UNDERLYING).balanceOf(
+      address(AaveV3Ethereum.COLLECTOR)
+    );
+    AaveV3Ethereum.COLLECTOR.transfer(
+      AaveV3EthereumAssets.rETH_UNDERLYING,
+      address(SWAPPER),
+      rethBalance
+    );
+
+    AaveV3Ethereum.COLLECTOR.transfer(
+      AaveV3EthereumAssets.wstETH_UNDERLYING,
+      address(AAVE_STETH_WITHDRAWER),
+      WSTETH_AMOUNT
+    );
+
+    uint256[] memory amounts = new uint256[](1);
+    amounts[0] = WSTETH_AMOUNT;
+
+    AAVE_STETH_WITHDRAWER.startWithdraw(amounts);
+
+    SWAPPER.swap(
+      MILKMAN,
+      PRICE_CHECKER,
+      AaveV3EthereumAssets.rETH_UNDERLYING,
+      AaveV3EthereumAssets.WETH_UNDERLYING,
+      AaveV3EthereumAssets.rETH_ORACLE,
```

Comment:
@defijesus This is also capped feed and chainlink should be used. Can you double check it for all the swaps.